lua/telescope/builtin/internal.lua

```diff
@@ -403,27 +403,33 @@ internal.help_tags = function(opts)
 end
 
 internal.man_pages = function(opts)
-  local pages = utils.get_os_command_output(opts.man_cmd or { 'apropos', '--sections=1', '' })
+  opts.sections = utils.get_default(opts.sections, {'1'})
+  assert(vim.tbl_islist(opts.sections), 'sections should be a list')
+  opts.man_cmd = utils.get_lazy_default(opts.man_cmd, function()
+    return vim.fn.has'mac' and {'apropos', ' '} or {'apropos', ''}
```

Comment:
A vimL free version to check if mac would be: `vim.loop.os_uname().sysname == 'Darwin'`

Reply:
Oh, good stuff. I did not know this. ee5c8f0